### ISSUE TITLE
Scheduler ical library needs updating

### ISSUE THREAD

**Issue by gregorydlogan** [opencast/opencast#1124]
Per #712, we're currently using a patched iCal4j library.  The fixes Plapadoo filed against iCal4j have been merged (https://github.com/ical4j/ical4j/pull/314), so we should update the library.

In doing this, it might make sense to create a DST unit test in the scheduler service as well.  We currently have some from the index (per #712), but I don't see any in the scheduler.

**Comment by lkiesow** [comment]
Careful with this. If I remember correctly, the OSGI loader for this library got patched as well.

**Comment by gregorydlogan** [comment]
I believe you might be correct.  This is also why I did *not* mark this as an easy issue :)

**Comment by krinnewitz** [comment]
Yes, I can confirm that only one of the two fixes got merged. OSGI-support is still missing.
